gammapy/spectrum/models.py

```diff
@@ -1063,31 +1065,174 @@ def plot(self, energy_range, ax=None, energy_unit='TeV',
         return ax
 
 
-class AbsorbedSpectralModel(SpectralModel):
+class Absorption(object):
+    """
+    Class dealing with absorption model
+
+    Parameters
+    ----------
+    energy_lo : `~astropy.units.Quantity`
+        Lower bin edges of energy axis
+    energy_hi : `~astropy.units.Quantity`
+        Upper bin edges of energy axis
+    param_lo : `~astropy.units.Quantity`
+        Lower bin edges of parameter axis
+    param_hi : `~astropy.units.Quantity`
+        Upper bin edges of parameter axis
+    data : `~astropy.units.Quantity`
+        Model value
+
+    Examples
+    --------
+    Create and plot EBL absorption model for a redshift of 0.2
+
+    .. plot::
+        :include-source:
+        import matplotlib.pyplot as plt
+        from gammapy.spectrum.models import Absorption
 
-    def __init__(self, spectral_model, table_model):
-        """Absorbed spectral model
+        table = Absorption.read(filename='$GAMMAPY_EXTRA/datasets/ebl/ebl_dominguez11.fits.gz').table_model(parameter=0.2)
+
+        ax = plt.gca()
+        opts = dict(energy_range=[0.01, 100] * u.TeV, energy_unit='TeV', ax=ax)
```

Comment:
Does this example work?
You're missing an import `import astropy.units as u`, no?

What I do to test examples locally is to copying the code snipped and then typing `%paste` and enter in IPython.